src/Compilers/Core/Portable/Operations/IVariableDeclarationOperation.cs

```diff
@@ -42,5 +42,11 @@ public interface IVariableDeclarationOperation : IOperation
         /// In C#, this will always be null.
         /// </remarks>
         IVariableInitializerOperation Initializer { get; }
+
+        /// <summary>
+        /// Array dimensions supplied to an array declaration in error cases, ignored by the compiler. This is only used for the C# case of
+        /// RankSpecifierSyntax nodes on an ArrayTypeSyntax.
+        /// </summary>
+        ImmutableArray<IOperation> IgnoredDimensions { get; }
```

Comment:
>ImmutableArray<IOperation> IgnoredDimensions { get; } [](start = 8, length = 53)

It looks like we already expose this information through IVariableDeclaratorOperation.IgnoredArguments

Reply:
That's for 
int X[10]

Not
int[] x